Framework/Algorithms/test/CreateTransmissionWorkspace2Test.h

```diff
@@ -363,6 +363,30 @@ class CreateTransmissionWorkspace2Test : public CxxTest::TestSuite {
     TS_ASSERT_DELTA(outLam->x(0)[3], 2.0924, 0.0001);
   }
 
+  void test_two_transmission_runs_stitch_ScaleRHSWorkspace() {
+
+    CreateTransmissionWorkspace2 alg;
+    alg.setChild(true);
+    alg.initialize();
+    alg.setProperty("FirstTransmissionRun", m_multiDetectorWS);
+    alg.setProperty("SecondTransmissionRun", m_multiDetectorWS);
+    alg.setProperty("WavelengthMin", 1.5);
+    alg.setProperty("WavelengthMax", 15.0);
+    alg.setPropertyValue("ProcessingInstructions", "2");
+    alg.setPropertyValue("OutputWorkspace", "outWS");
+    alg.setProperty("ScaleRHSWorkspace", false);
+    alg.execute();
+    MatrixWorkspace_sptr outLam = alg.getProperty("OutputWorkspace");
+
+    TS_ASSERT_EQUALS(outLam->getNumberHistograms(), 1);
+    TS_ASSERT_EQUALS(outLam->blocksize(), 14);
+    TS_ASSERT(outLam->x(0)[0] >= 1.5);
+    TS_ASSERT(outLam->x(0)[14] <= 15.0);
+    TS_ASSERT_DELTA(outLam->y(0)[0], 2.000, 0.0001);
```

Comment:
Why is the y-data `2.0`? 

Also, Are we expecting counts or frequencies? Maybe go explicitly to `outLam->counts()` or `outLam->frequencies()` to get data.